spec/comment_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'Validations For the Comment model' do
+    before(:each) do
+      @user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.',
+                          posts_counter: 0)
```

Comment:
@- [ ] Dear @fpsapc, Please remember all custom methods in the models should have a corresponding unit test. And for the `Comment` model we need to test the `update_post_comments_counter` method.